ebml_matroska.xml

```diff
@@ -298,15 +298,35 @@ This **SHOULD** be kept the same when making a direct stream copy of the Track t
-    <documentation lang="en" purpose="definition">Set if that track (audio, video or subs) **SHOULD** be active if no language found matches the user preference. (1 bit)</documentation>
+    <documentation lang="en" purpose="definition">Set if that track (audio, video or subs) **SHOULD** be eligible for automatic selection by the player. (1 bit)</documentation>
     <extension cppname="TrackFlagDefault"/>
   </element>
   <element name="FlagForced" path="\Segment\Tracks\TrackEntry\FlagForced" id="0x55AA" type="uinteger" range="0-1" default="0" minOccurs="1" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Set if that track **MUST** be active during playback. There can be many forced track for a kind (audio, video or subs),
-the player **SHOULD** select the one which language matches the user preference or the default + forced track.
-Overlay **MAY** happen between a forced and non-forced track of the same kind. (1 bit)</documentation>
+    <documentation lang="en" purpose="definition">Applies only to subtitles. Set if that track SHOULD be active during playback by default if it matches the user's language preference,
+even if the user's preferences would normally not enable subtitles with the selected audio track;
+this can be used for tracks containing only translations of foreign-language audio or onscreen text. (1 bit)</documentation>
     <extension cppname="TrackFlagForced"/>
   </element>
+  <element name="FlagHearingImpaired" path="\Segment\Tracks\TrackEntry\FlagHearingImpaired" id="0x55AB" type="uinteger" range="0-1" default="0" minOccurs="0" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set if that track is suitable for users with hearing impairments. (1 bit)</documentation>
+    <extension cppname="TrackFlagHearingImpaired"/>
+  </element>
+  <element name="FlagVisualImpaired" path="\Segment\Tracks\TrackEntry\FlagVisualImpaired" id="0x55AC" type="uinteger" range="0-1" default="0" minOccurs="0" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set if that track is suitable for users with visual impairments. (1 bit)</documentation>
+    <extension cppname="TrackFlagVisualImpaired"/>
+  </element>
+  <element name="FlagDescriptions" path="\Segment\Tracks\TrackEntry\FlagDescriptions" id="0x55AD" type="uinteger" range="0-1" default="0" minOccurs="0" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set if that track contains textual descriptions of video content. (1 bit)</documentation>
+    <extension cppname="TrackFlagDescriptions"/>
+  </element>
+  <element name="FlagOriginal" path="\Segment\Tracks\TrackEntry\FlagOriginal" id="0x55AE" type="uinteger" range="0-1" default="0" minOccurs="0" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set if that track is in the content's original language (not a translation). (1 bit)</documentation>
```

Comment:
I think that's kind of the `FlagDefault` as well. It may actually be a saner description for `FlagDefault`, which explains why it should be used in absence of user preferred language (or if the user prefers the default language in general, like me).

Reply:
The issue is that a sizable portion of existing files don't use `FlagDefault` that way (i.e. they set it on a track that's _not_ in the original language); the new description of `FlagDefault` is designed to provide reasonable behavior even with existing files. The use I imagine for it is "provides regular service in a given language" (as opposed to any of the features that one would need to opt-in to get auto-selected, like descriptive audio or commentary or any other possible extended service), and there may be multiple such tracks per language (e.g. a 5.1 and a 2.0 track are reasonably common, or sometimes even 2 different codecs with the same channel layout).

Reply:
Even a new flag may be misused so existing files may not be the right reference. Especially if some muxing tools just automatically set it on the first track or something.
On the other hand the fact there may be multiple versions of the same language track is definitely a good point.